src/components/PoolOverview.tsx

```diff
@@ -86,21 +85,22 @@ export default function PoolOverview({
       <div className="left">
         <div className="titleAndTag">
           {isMetapool ? (
-            <ToolTip content={t("metapool")}>
+            <Tooltip title={<React.Fragment>{t("metapool")}</React.Fragment>}>
```

Comment:
Does it work without fragment here?

Reply:
Don't need <React.Fragment>. Because `t("metapool") couldn't be `undefined, so I used <React.Fragment>
But we can use `<Tooltip title={t("metapool") || ""}>` alternatively.

Reply:
I believe it will fallback to the English translation by default so the `|| ""` may be unnecessary.

Reply:
![image](https://user-images.githubusercontent.com/75422800/157430406-c5a5a590-7b04-497a-a83d-856f9e92e3f7.png)
Does it mean that `TFunctionResult` could have `undefined`?

Reply:
That is odd. In this case can you just do `title={t("metapool") || "Metapool"` ?

Reply:
OK